
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Lecutre 25: Genome assembly &#8212; bioinformatics@nau</title>
    <link rel="stylesheet" href="../../../static/pygments.css" type="text/css" />
    <link rel="stylesheet" href="../../../static/haiku.css" type="text/css" />
    <script id="documentation_options" data-url_root="../../../" src="../../../static/documentation_options.js"></script>
    <script src="../../../static/jquery.js"></script>
    <script src="../../../static/underscore.js"></script>
    <script src="../../../static/doctools.js"></script>
    <link rel="search" title="Search" href="../../../search.html" />
    <link rel="next" title="Lecture 26: Genome assembly and annotation" href="lecture26.html" />
    <link rel="prev" title="Lecutre 23: Sequence covariation and methods comparisons in bioinformatics" href="lecture23.html" /> 
  </head><body>
      <div class="header" role="banner"><h1 class="heading"><a href="../../../index.html">
          <span>bioinformatics@nau</span></a></h1>
        <h2 class="heading"><span>Lecutre 25: Genome assembly</span></h2>
      </div>
      <div class="topnav" role="navigation" aria-label="top navigation">
      
        <p>
        «&#160;&#160;<a href="lecture23.html">Lecutre 23: Sequence covariation and methods comparisons in bioinformatics</a>
        &#160;&#160;::&#160;&#160;
        <a class="uplink" href="../../../index.html">Contents</a>
        &#160;&#160;::&#160;&#160;
        <a href="lecture26.html">Lecture 26: Genome assembly and annotation</a>&#160;&#160;»
        </p>

      </div>
      <div class="content" role="main">
        
        
  <div class="section" id="lecutre-25-genome-assembly">
<h1>Lecutre 25: Genome assembly<a class="headerlink" href="#lecutre-25-genome-assembly" title="Permalink to this headline">¶</a></h1>
<dl class="simple">
<dt>What is genome sequencing?</dt><dd><ul class="simple">
<li><p>The determination of the full order of bases (As, Cs, Gs, and Ts) in an organism’s genome.</p></li>
</ul>
</dd>
<dt>How do we sequence genomes?</dt><dd><ul class="simple">
<li><p>Current DNA sequencing technologies generate sequencing reads that are much shorter than the full genomic sequences we’re interested in. We therefore must sequence fragments of the genome and <code class="docutils literal notranslate"><span class="pre">assemble</span></code> them into the full length sequence of interest.</p></li>
<li><p>To achieve this, we must sequence in great depth so we observe each base multiple times. During the human genome project, the human genome was sequenced to <cite>8x</cite> coverage, meaning that on average every base was sequenced 8 times. This can help with sequencing error detection and correction, but also provides many overlapping reads which are necessary for assembly.</p></li>
</ul>
</dd>
<dt>Why is assembly hard?</dt><dd><ul class="simple">
<li><p><cite>Repeat elements</cite>, or fragments of DNA with identical sequences, can make a genome appear to fit together in multiple different ways. When the same sequence is observed many times, an assmebler (or program that performs genome assembly) will generally tag that sequence as a repeat.</p></li>
<li><p>Very short reads cause more of a repeat problem than longer reads. As you get shorter, repeats are more likely; as you get longer repeats are less likely. Note that genome sequence is not random though, so the probability of a repeat doesn’t just decrease exponentially with increasing read length (i.e., probability of a repeat is not 0.25**read_length).</p></li>
<li><p>Good discussion of the problems with assembly from short reads can be found <a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/20508146">here</a>.</p></li>
</ul>
</dd>
<dt>How do we assemble genomes:</dt><dd><ul class="simple">
<li><p>Create <cite>contigs</cite> by identifying overlapping reads segments and merging into contiguous segments.</p></li>
<li><p>In any complex genome, contigs will be interrupted by repeat regions or regions that do not get sequenced by chance. These can be assembled using <cite>paired end</cite> or <cite>mate pair</cite> reads. This involves sequencing both ends of sequences of known length, so for certain pairs of reads which would be incorporated into contigs, you’d know the length of sequence separating those reads. You can use this information to create <cite>scaffolds</cite>, which are contigs that are interrupted by gaps that represent unknown sequence.</p></li>
<li><p>Description of different approaches for genome assembly can be found <a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/20211242">here</a>.</p></li>
</ul>
</dd>
<dt>Finished versus draft genomes:</dt><dd><ul class="simple">
<li><p>Finished genome requires the full sequencing of the entire genome; draft allows small gaps due to missing coverage or segments that cannot be accurately determined due to repeat elements.</p></li>
<li><p>Generating the draft is a lot cheaper than generating the finished genome (the former is heavily automated, while the latter is not) so many projects stop at the draft.</p></li>
</ul>
</dd>
</dl>
</div>


      </div>
      <div class="bottomnav" role="navigation" aria-label="bottom navigation">
      
        <p>
        «&#160;&#160;<a href="lecture23.html">Lecutre 23: Sequence covariation and methods comparisons in bioinformatics</a>
        &#160;&#160;::&#160;&#160;
        <a class="uplink" href="../../../index.html">Contents</a>
        &#160;&#160;::&#160;&#160;
        <a href="lecture26.html">Lecture 26: Genome assembly and annotation</a>&#160;&#160;»
        </p>

      </div>

    <div class="footer" role="contentinfo">
        &#169; Copyright 2019, J Gregory Caporaso.
      Last updated on Jan 12, 2021.
      Created using <a href="https://www.sphinx-doc.org/">Sphinx</a> 3.4.3.
    </div>
  </body>
</html>
